sample/ios-app/src/TestCollectionViewController.swift

```diff
@@ -0,0 +1,43 @@
+/*
+* Copyright 2019 IceRock MAG Inc. Use of this source code is governed by the Apache 2.0 license.
+*/
+
+import UIKit
+import MultiPlatformLibrary
+import MultiPlatformLibraryUnits
+
+class TestCollectionViewController: UIViewController {
+    
+    @IBOutlet private var collectionView: UICollectionView!
+    private var dataSource: UICollectionViewDataSource!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        let testing = TestingCollection(unitFactory: self)
+
+        let dataSource = FlatUnitCollectionViewDataSource()
```

Comment:
maybe use UnitCollectionViewDataSource?

Reply:
https://github.com/icerockdev/moko-units/blob/40773b513d22094bf6f2ed3c7410e6d58c39b6f8/units/src/iosMain/kotlin/dev/icerock/moko/units/UnitCollectionViewDataSource.kt#L17

Reply:
UnitTableViewDataSource and UnitCollectionViewDataSource are unavailable from Swift-native side

MultiPlatformLibrary.h

```
__attribute__((unavailable("Kotlin subclass of Objective-C class can't be imported")))
__attribute__((swift_name("UnitCollectionViewDataSource")))
@interface MPLUnitCollectionViewDataSource : NSObject
@end;

__attribute__((unavailable("Kotlin subclass of Objective-C class can't be imported")))
__attribute__((swift_name("UnitTableViewDataSource")))
@interface MPLUnitTableViewDataSource : NSObject
@end;

```